index.html

```diff
@@ -41,8 +41,167 @@ <h1 class="primary">
+                    <div class="project-details">
+                        <h3 class="project-title"> Multi-Post Stories <br> Gain+Glory</h3>
+                    
+                        <ul class="tech-stack">
+                            <li class="tag">Ruby on Rails</li>
+                            <li class="tag">CSS</li>
+                            <li class="tag">JavaScript</li>
+                            <li class="tag">HTML</li> 
+                        </ul>
+                        <div class="btn-container">
+                            <button class="see-project-btn" type="button">See project</button>
+                        </div>
+                    </div>
+                </div>
+                <div class="project-container">
+                    <div class="img-container"></div>
+                    <div class="project-details">
+                        <h3 class="project-title"> Multi-Post Stories <br> Gain+Glory</h3>
+                    
+                        <ul class="tech-stack">
+                            <li class="tag">Ruby on Rails</li>
+                            <li class="tag">CSS</li>
+                            <li class="tag">JavaScript</li>
+                            <li class="tag">HTML</li> 
+                        </ul>
+                        <div class="btn-container">
+                            <button class="see-project-btn" type="button">See project</button>
+                        </div>
+                    </div>
+                </div>
+                <div class="project-container">
+                    <div class="img-container"></div>
+                    <div class="project-details">
+                        <h3 class="project-title"> Multi-Post Stories <br> Gain+Glory</h3>
+                    
+                        <ul class="tech-stack">
+                            <li class="tag">Ruby on Rails</li>
+                            <li class="tag">CSS</li>
+                            <li class="tag">JavaScript</li>
+                            <li class="tag">HTML</li> 
+                        </ul>
+                        <div class="btn-container">
+                            <button class="see-project-btn" type="button">See project</button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <section class="heading">
+              <h1 class="primary color1">About me</h1>  
+              <p class="supporting-text"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hesitate to contact me.
+              </p>
+              <div class="btn-container">
+                <button class="resume-btn">Get my resume</button>
+              </div>
+        </section>
+
+        <section class="qualifications" >
```

Comment:
Kindly make sure you use the `section` tag only for the main section of a container.
I suggest you use the `div` tag instead of the `section` tag here to follow HTML best practices. 😄